saleor/graphql/discount/mutations/voucher/voucher_create.py

```diff
@@ -255,9 +259,16 @@ def save(  # type: ignore[override]
             models.VoucherCode.objects.bulk_create(code_instances)
 
     @classmethod
-    def post_save_action(cls, info: ResolveInfo, instance, code):
+    def post_save_action(  # type: ignore[override]
+        cls, info: ResolveInfo, instance, codes_instances, cleaned_input
+    ):
         manager = get_plugin_manager_promise(info.context).get()
-        cls.call_event(manager.voucher_created, instance, code)
+        cls.call_event(manager.voucher_created, instance, codes_instances[-1].code)
+        if codes_instances:
```

Comment:
This is strange. In line 266, you are making: `codes_instances[-1].code`, and this means for me that `codes_instances` always has at least single code. 
But line later 267, you have `if` condition to check if `code_instances` has some objects. 
Which one is true?

Reply:
fixed